portfolio/src/main/webapp/script.js

```diff
@@ -115,28 +118,41 @@ function createCommentElement(comment) {
 }
 
 /** Tells the server to delete the task. */
-function deleteTask(comment) {
+function deleteTask(commentId) {
   const params = new URLSearchParams();
-  params.append('id', comment.id);
+  params.append('id', commentId);
   fetch('/delete-comment', {method: 'POST', body: params});
+  window.location.replace("index.html");
 }
 
-/* When the user clicks on the button, 
-toggle between hiding and showing the dropdown content */
 function commentSettings() {
-  document.getElementById("myDropdown").classList.toggle("show");
+  document.getElementById("dropdown-content").style.visibility = "visible"; 
+  document.getElementById("dropdown-content").style.display = "inline-block"; 
+}  
+
+function sendForm() {
+  window.location.replace("index.html?maxcomments="+ document.getElementById('quantity').value );
 }
 
-// Close the dropdown if the user clicks outside of it
-window.onclick = function(event) {
-  if (!event.target.matches('.commentSettings')) {
-    var dropdowns = document.getElementsByClassName("dropdown-content");
-    var i;
-    for (i = 0; i < dropdowns.length; i++) {
-      var openDropdown = dropdowns[i];
-      if (openDropdown.classList.contains('show')) {
-        openDropdown.classList.remove('show');
-      }
-    }
-  }
+function getMaxComments() {
+    const defaultValue = 15;
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    if(urlParams.get('maxComments') == null)
+        return defaultValue;
+
+    else
+        return(urlParams.get('maxComments'));
+}
+
+function fetchBlobstoreUrlAndShowForm() {
+  fetch('/blobstore-upload-url')
+      .then((response) => {
+        return response.text();
+      })
```

Comment:
This lambda function could be written shorter: `(response) => response.text()`

Reply:
Oh yes! 
Done.